CSharpBasics/FinalTest/Tests/TrainingTests.cs

```diff
@@ -0,0 +1,39 @@
+using NUnit.Framework;
+
+namespace FinalTest.Tests
+{
+    [TestFixture]
+    public class TrainingTests : BaseTest
+    {
+        private TrainingMainPage MainPage;
+
+        private const string _mainLinkTrainingPage = "https://training.by/";
+
+
+        [SetUp]
+        public void Initialaze()
+        {
+            MainPage = new TrainingMainPage();
+        }
+
+        [TestCase(Languages.ENGLISH)]
+        [TestCase(Languages.�������)]
+        [TestCase(Languages.���������)]
+        public void CheckCorrectPageIsOpened(Languages language)
+        {
+            Driver.Navigate().GoToUrl(_mainLinkTrainingPage);
+
+            Driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(10);
+
+            MainPage.OpenLanguagesDropDown();
+
+            MainPage.SelectLanguageInDropDown(language);
+
+            Thread.Sleep(2000);
```

Comment:
you can wait till some element on the page is displayed. For example, hero banner on the main page.